keyboards/nopunin10did/railroad/info.json

```diff
@@ -0,0 +1,125 @@
+                {"y":1, "x":10.25, "label":"F2"}, 
+                {"y":1, "x":11.25, "label":"F6"},        
+                {"y":1, "x":12.25, "label":"F10"},
+                {"y":2, "x":10.25, "label":"F3"}, 
+                {"y":2, "x":11.25, "label":"F7"},        
+                {"y":2, "x":12.25, "label":"F11"}, 
+                {"y":3, "x":10.25, "label":"F4"}, 
+                {"y":3, "x":11.25, "label":"F8"},        
+                {"y":3, "x":12.25, "label":"F12"},
+                
+                {"y":0, "x":13.5, "label":"="},
+                {"y":0, "x":14.5, "label":"6"},
+                {"y":0, "x":15.5, "label":"7"},
+                {"y":0, "x":16.5, "label":"8"},
+                {"y":0, "x":17.5, "label":"9"},
+                {"y":0, "x":18.5, "label":"0"},
+                {"y":0, "x":19.5, "label":"Backspace", "w":2},
+                {"y":0, "x":21.5, "label":"Ins"},
+                {"y":0, "x":22.5, "label":"Esc"},
+                
+                {"y":1, "x":13.5, "label":"]"},
+                {"y":1, "x":14.5, "label":"Y"},
+                {"y":1, "x":15.5, "label":"U"},
+                {"y":1, "x":16.5, "label":"I"},
+                {"y":1, "x":17.5, "label":"O"},
+                {"y":1, "x":18.5, "label":"P"},
+                {"y":1, "x":19.5, "label":"\\", "w":1.5},
+                {"y":1, "x":21,   "label":"Del"},
+                
+                {"y":2, "x":13.5, "label":"Home"},
+                {"y":2, "x":14.5, "label":"H"},
+                {"y":2, "x":15.5, "label":"J"},
+                {"y":2, "x":16.5, "label":"K"},
+                {"y":2, "x":17.5, "label":"L"},
+                {"y":2, "x":18.5, "label":";"},
+                {"y":2, "x":19.5, "label":"'"},
+                {"y":2, "x":20.5, "label":"Return", "w":1.75},
+                
+                {"y":1, "x":22, "label":"ISO Enter", 
+                    "ks":[[0,0], [1.5,0], [1.5,2], [0.25,2], [0.25,1], [0,1], [0,0]]},
+                
+                {"y":3, "x":13.5,    "label":"End"},
+                {"y":3, "x":14.5,    "label":"N"},
+                {"y":3, "x":15.5,    "label":"M"},
+                {"y":3, "x":16.5,    "label":","},
+                {"y":3, "x":17.5,    "label":"."},
+                {"y":3, "x":18.5,    "label":"/"},
+                {"y":3, "x":19.5,    "label":"Shift", "w":1.75},
+                {"y":3.25, "x":21.5, "label":"Up"},
+                
+                {"y":4.25, "x":12,    "label":"LShftAlt / Calc", "w":2.25},
+                {"y":4.25, "x":14.25, "label":"Space", "w":1.75},
+                {"y":4.25, "x":16,    "label":"UK #~"},
+                {"y":4, "x":17.25,    "label":"AltGr / Home", "w":1.5},
+                {"y":4, "x":17.25,    "label":"Ctrl / End", "w":1.5},
+                {"y":4.25, "x":20.5,  "label":"Left"},
+                {"y":4.25, "x":21.5,  "label":"Down"},
+                {"y":4.25, "x":22.5,  "label":"Right"}
```

Comment:
This file needs to use the same key ordering that `keymap.c` uses in order to work correctly.

```suggestion
                {"y":0, "x":0, "label":"Scroll"},
                {"y":0, "x":1, "label":"Pause"},
                {"y":0, "x":2, "label":"Esc"},
                {"y":0, "x":3, "label":"`"},
                {"y":0, "x":4, "label":"1"},
                {"y":0, "x":5, "label":"2"},
                {"y":0, "x":6, "label":"3"},
                {"y":0, "x":7, "label":"4"},
                {"y":0, "x":8, "label":"5"},
                {"y":0, "x":9, "label":"-"},

                {"y":0, "x":10.25, "label":"F1"},
                {"y":0, "x":11.25, "label":"F5"},
                {"y":0, "x":12.25, "label":"F9"},

                {"y":0, "x":13.5, "label":"="},
                {"y":0, "x":14.5, "label":"6"},
                {"y":0, "x":15.5, "label":"7"},
                {"y":0, "x":16.5, "label":"8"},
                {"y":0, "x":17.5, "label":"9"},
                {"y":0, "x":18.5, "label":"0"},
                {"y":0, "x":19.5, "label":"Backspace", "w":2},
                {"y":0, "x":21.5, "label":"Ins"},
                {"y":0, "x":22.5, "label":"Esc"},

                {"y":1, "x":0,   "label":"Print"},
                {"y":1, "x":1,   "label":"Backspace", "w":1.5},
                {"y":1, "x":2.5, "label":"Tab", "w":1.5},
                {"y":1, "x":4,   "label":"Q"},
                {"y":1, "x":5,   "label":"W"},
                {"y":1, "x":6,   "label":"E"},
                {"y":1, "x":7,   "label":"R"},
                {"y":1, "x":8,   "label":"T"},
                {"y":1, "x":9,   "label":"["},

                {"y":1, "x":10.25, "label":"F2"},
                {"y":1, "x":11.25, "label":"F6"},
                {"y":1, "x":12.25, "label":"F10"},

                {"y":1, "x":13.5, "label":"]"},
                {"y":1, "x":14.5, "label":"Y"},
                {"y":1, "x":15.5, "label":"U"},
                {"y":1, "x":16.5, "label":"I"},
                {"y":1, "x":17.5, "label":"O"},
                {"y":1, "x":18.5, "label":"P"},
                {"y":1, "x":19.5, "label":"\\", "w":1.5},
                {"y":1, "x":21,   "label":"Del"},

                {"y":2, "x":0,    "label":"Caps", "w":1.25},
                {"y":2, "x":1.75, "label":"Control", "w":2.25},
                {"y":2, "x":4,    "label":"A"},
                {"y":2, "x":5,    "label":"S"},
                {"y":2, "x":6,    "label":"D"},
                {"y":2, "x":7,    "label":"F"},
                {"y":2, "x":8,    "label":"G"},
                {"y":2, "x":9,    "label":"PgUp"},

                {"y":2, "x":10.25, "label":"F3"},
                {"y":2, "x":11.25, "label":"F7"},
                {"y":2, "x":12.25, "label":"F11"},

                {"y":2, "x":13.5, "label":"Home"},
                {"y":2, "x":14.5, "label":"H"},
                {"y":2, "x":15.5, "label":"J"},
                {"y":2, "x":16.5, "label":"K"},
                {"y":2, "x":17.5, "label":"L"},
                {"y":2, "x":18.5, "label":";"},
                {"y":2, "x":19.5, "label":"'"},
                {"y":2, "x":20.5, "label":"Return", "w":1.75},

                {"y":1, "x":22.25, "label":"ISO Enter", "w":1.25, "h":2},

                {"y":3.25, "x":0, "label":"Cut"},
                {"y":3,    "x":1.25, "label":"Shift", "w":2.75},
                {"y":3,    "x":4, "label":"Z"},
                {"y":3,    "x":5, "label":"X"},
                {"y":3,    "x":6, "label":"C"},
                {"y":3,    "x":7, "label":"V"},
                {"y":3,    "x":8, "label":"B"},
                {"y":3,    "x":9, "label":"PgDn"},

                {"y":3, "x":10.25, "label":"F4"},
                {"y":3, "x":11.25, "label":"F8"},
                {"y":3, "x":12.25, "label":"F12"},

                {"y":3, "x":13.5,    "label":"End"},
                {"y":3, "x":14.5,    "label":"N"},
                {"y":3, "x":15.5,    "label":"M"},
                {"y":3, "x":16.5,    "label":","},
                {"y":3, "x":17.5,    "label":"."},
                {"y":3, "x":18.5,    "label":"/"},
                {"y":3, "x":19.5,    "label":"Shift", "w":1.75},
                {"y":3.25, "x":21.5, "label":"Up"},

                {"y":4.25, "x":0,    "label":"Copy"},
                {"y":4.25, "x":1,    "label":"Paste"},
                {"y":4, "x":2.25,    "label":"Control", "w":1.5},
                {"y":4, "x":3.75,    "label":"Super", "w":1.25},
                {"y":4, "x":5,       "label":"Alt", "w":1.25},
                {"y":4.25, "x":6.5,  "label":"UK \\|"},
                {"y":4.25, "x":7.5,  "label":"Space", "w":1.75},
                {"y":4.25, "x":9.25, "label":"Shift / Del", "w":2.75},

                {"y":4.25, "x":12,    "label":"LShftAlt / Calc", "w":2.25},
                {"y":4.25, "x":14.25, "label":"Space", "w":1.75},
                {"y":4.25, "x":16,    "label":"UK #~"},
                {"y":4, "x":17.25,    "label":"AltGr / Home", "w":1.5},
                {"y":4, "x":18.75,    "label":"Ctrl / End", "w":1.5},
                {"y":4.25, "x":20.5,  "label":"Left"},
                {"y":4.25, "x":21.5,  "label":"Down"},
                {"y":4.25, "x":22.5,  "label":"Right"}
```

Reply:
It did use the same ordering as keymap.c.

There likely is a mistake with the one shift key.  If the ISO enter isn't rendering properly I can simplify it (though per the guide, I used the exact specification suggested there).

Reply:
I've corrected the position of the left shift and simplified the ISO enter.

Reply:
Context: Configurator's API Code actually doesn't know how to use the `ks` key, and over the years we've worked around that by just making it a 1.25x2u key.